ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,774 @@
+ enlisted to the transaction for purposes of this quickstart demonstration.
+
+|===
+
+== Setup the environment
+
+For the EJB remote calls works there is needed to setup all {productName} servers. Let's see
+what are necessary steps:
+
+It's needed to start three instances of application server.
+The easiest way to do so on local computer is to copy the {productName} installation directory
+to three separate directories.
+
+The first server (`server1`) is a caller &ndash; it contains an EJB which executes
+a remote call to one of the remote servers (`server2`,`server3`), which are callees.
+The distribution directory for the `server1` is reffered as `__{jbossHomeName}_1__`,
+for the `server2` it's `__{jbossHomeName}_2__` and for the `server3` it's `__{jbossHomeName}_3__`.
+
+=== Configure the credentials for remote call authentication
+
+For the remote call could be processed successfully from `server1` to either `server2`
+or to `server3` there has to be created a user at the calee side
+that the call will be authenticated with.
+
+[source,sh,subs="+quotes,attributes+"]
+----
+# go to the directory with distribution of server2 and server3
+# do the same for ${jbossHomeName}_3 as well
+cd ${jbossHomeName}_2
+# add user to 'ApplicationRealm' with command line tool
+./bin/add-user.sh -a -u ejb -p ejb -ds
+# with parameter -ds secret will be printed
+## To represent the user add the following to the server-identities definition <secret value="ZWpi" />
+----
+
+The user named `ejb` will be added to the `ApplicationRealm` with password `ejb`.
+The command line attribute `-ds` ensures that after the user is added
+there is printed the secret value that will be used in setting up
+the remote outbound connection at `server1`.
+
+Now the configuration of `server1` follows as set of JBoss CLI commands to be run.
+The cli script is configured to run with `server.xml` config file in embed mode
+(not needed to start the application server before the CLI is run).
+
+[source,sh,subs="+quotes,attributes+"]
+----
+# go to the directory with distribution of server1
+cd ${jbossHomeName}_1
+./bin/jboss-cli.sh \
+  --properties=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/remote-configuration.local.properties \
+  --file=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/remote-configuration.cli
+----
+
+=== Configure datasources
+
+As said for the transaction work could be done the quickstarts uses datasources.
+For that it's needed to have a database and configure the server to know how to connect to it.
+This quickstart works either with H2 embed database or PostgreSQL.
+
+==== Option 1: Running with embed H2 database
+
+The configuration of H2 is easy, well suited for local testing showcase
+but it skips the steps needed to be done for other databases
+and furthermore it's not suited for the OpenShift environment.
+
+For running the quickstart with H2 you need just rename the files
```

Comment:
*-ds.xml have been deprecated for long, is there no other option?

Reply:
I see. I can remove the `-ds.xml` and I'll add the steps for the datasource creation to the CLI script.